kubetest/main.go

```diff
@@ -274,6 +277,7 @@ func validateFlags(o *options) error {
 
 func main() {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
+	log.Printf("Running kubetest version: %s", gitTag)
```

Comment:
+1 to printing it always!

Additionally we can just exit after `validateFlags` (line 295) if `version` is true
i.e. when `kubetest --version` is invoked

Reply:
Returning on version makes sense. I think we should do it as a part of `run()`  on line 59 after printing the version

Reply:
I think the earlier we terminate, the better.
Usually `--version` would be requested explicitly and wouldn't be coupled with other flags like `--up` `--down` (and/or version gets priority) (similar to `--help`)

so simple invocations like`kubetest --version` can avoid going through `complete -> (acquireKubernetes/validWorkingDirectory) ->run`

Reply:
Oh. I wanted to keep the handler for version command inside of `run()` for code consistency.
But it makes sense to return early. Will inline this right after `validateFlags`